src/server/server.cc

```diff
@@ -245,6 +244,9 @@ void Server::Join() {
   if (auto s = task_runner_.Join(); !s) {
     LOG(WARNING) << s.Msg();
   }
+  for (const auto &worker : worker_threads_) {
```

Comment:
Just curious, why worker would shutdown after the other threads shutdown?

Reply:
It doesn't wait for other threads but can't be the first one, because worker_threads_ may be modified after starting, so I move it to the last.